packages/extender/src/registry/registry.js

```diff
@@ -0,0 +1,69 @@
+import * as _ from 'lodash';
+import {composeServices} from './composeServices';
+
+class Registry {
+    constructor() {
+        this.registry = {};
+    }
+
+    addOrReplace(type, key) {
+        const registryKey = type + '-' + key;
+
+        let actions = Array.prototype.slice.call(arguments, 2);
+        let action = composeServices(...actions);
```

Comment:
```suggestion
        const action = composeServices(...actions);
```
Je suppose que ça doit passer :)